### ISSUE TITLE
Support ruby 2.2

### ISSUE THREAD

**Issue by sue445** [tslocke/rubydoctest#16]
`v1.1.3` does not work on ruby 2.2 because `Config` is removed.
But this was fixed at 503beea040252bdaa1d8bea6bbe18974058c30b9

Please release new version :pray: 
## example

``` bash
$ ruby -v
ruby 2.2.0preview2 (2014-11-28 trunk 48628) [x86_64-darwin13]

$ bundle exec rubydoctest README.md
/Users/sue445/dev/workspace/github.com/sue445/date_discreter/vendor/bundle/ruby/2.2.0/gems/rubydoctest-1.1.3/bin/rubydoctest:4:in `<top (required)>': uninitialized constant Config (NameError)
    from /Users/sue445/dev/workspace/github.com/sue445/date_discreter/vendor/bundle/ruby/2.2.0/bin/rubydoctest:23:in `load'
    from /Users/sue445/dev/workspace/github.com/sue445/date_discreter/vendor/bundle/ruby/2.2.0/bin/rubydoctest:23:in `<main>'
```

Please release new version

**Comment by canadaduane** [comment]
Released as 1.1.5

**Comment by sue445** [comment]
:tada: :tada: :tada: :tada:
